tests/console/force_cron_run.pm

```diff
@@ -25,7 +25,7 @@ sub settle_load {
     # JeOS is different to SLE general as it extends the appliance's disk on first boot,
     # so the balance is a different challenge to SLE. Elapsed time is not necessary a key
     # measure here, responsiveness of the system is.
-    record_soft_failure 'bsc#1063638' if (time - $before) > (is_jeos() ? 180 : 30);
+    record_soft_failure 'bsc#1063638' if (time - $before) > (is_jeos() ? 180 : 70);
```

Comment:
To be honest, as it's soft-failure for known issue I would even put same timeout for both, unless we don't know why it became an issue for SLE recently.

Reply:
https://progress.opensuse.org/issues/25554, comment 12. 
I discussed a little bit with okurz, the further test runs on e13.suse.de shows quite different results, some passed, some still softfailed. But this is in general a compromise to see that is not so strict as before.
Sure I can timeout value for both and check this again...

Reply:
So it might make sence to use 180 for both, anyway